src/polymer-cli.ts

```diff
@@ -42,6 +43,21 @@ export class PolymerCli {
   }
 
   run(args) {
+    // If the "--version"/"-v" flag is ever present, just print
```

Comment:
I know, me too. It's mostly to work around limitations in our arguments parser & architecture. If we didn't do this, we'd have to add `--help` support to each command individually (otherwise the parser would throw an error [a few lines below here](https://github.com/Polymer/polymer-cli/pull/65/files#diff-81f7c55176652a78582250c2c66fbd3aR62)). Then we'd need logic inside of each command to bounce us out and into the help command. 

In the end this seemed like the cleanest choice until we can reevaluate our args parser and/or need to move between commands.